build.sc

```diff
@@ -13,7 +13,7 @@ import de.tobiasroeser.mill.vcs.version.VcsVersion
 import $ivy.`com.github.lolgab::mill-mima::0.0.13`
 import com.github.lolgab.mill.mima._
 
-val scala31 = "3.1.3"
```

Comment:
Do you drop Scala 3.1 support on purpose?

Reply:
Yeah, 3.1 gives some weird `unable to call macro trait in same file` errors that go away on 3.22